src/lib/date-picker/month-view.ts

```diff
@@ -0,0 +1,119 @@
+import {
+  Component,
+  ViewEncapsulation,
+  ChangeDetectionStrategy,
+  Input,
+  EventEmitter,
+  Output,
+  AfterContentInit
+} from '@angular/core';
+import {coerceDateProperty} from '../core/coercion/date-property';
+import {MdCalendarCell} from './calendar-table';
+import {DateLocale} from './date-locale';
+
+
+const DAYS_PER_WEEK = 7;
```

Comment:
I'm not 100% sure because the world is a crazy place, but I think this is universally true and doesn't necessarily need to be put in a constant (e.g., you wouldn't make a constant like `ONE_HUNDRED`).